src/frontends/lean/parser.cpp

```diff
@@ -1425,14 +1425,15 @@ static pair<notation::transition, parse_table> const * get_non_skip(list<pair<no
 expr parser::parse_notation(parse_table t, expr * left) {
     check_system("parse_notation");
     lean_assert(curr() == token_kind::Keyword);
-    auto p = pos();
+    auto cur_pos = pos();
+    auto p = left ? expr_ast(*left).m_start : cur_pos;
```

Comment:
I don't understand this change; I'm probably blind, but it appears that nothing uses `p` below any more?